boards/seeeduino_xiao/doc.txt

```diff
@@ -0,0 +1,38 @@
+/**
+@defgroup    boards_seeeduino_XIAO Seeeduino_XIAO
+@ingroup     boards
+@brief       Support for the Seeeduino XIAO.
+
+### General information
+
+The Seeeduino XIAO is the smallest member of the Seeeduino family. It carries the powerful ATSAMD21G18A-MU which is a low-power microcontrollers. On the other hand, this little board has good performance in processing but needs less power. As a matter of fact, it is designed in a tiny size and can be used for wearable devices and small projects.
+
+Seeeduino XIAO has 14 GPIO PINs, which can be used for 11 digital interfaces, 11 mock interfaces, 10 PWM interfaces (d1-d10), 1 DAC output pin D0, 1 SWD pad interface, 1 I2C interface, 1 SPI interface, 1 UART interface, Serial communication indicator (T/R), Blink light (L). The colors of LEDs(Power,L,RX,TX) are green, yellow, blue and blue. Moreover, Seeeduino XIAO has a Type-C interface which can supply power and download code. There are two reset button, you can short connect them to reset the board.
+
+-- General description of the [wiki](https://wiki.seeedstudio.com/Seeeduino-XIAO/)
+
+### Pinout
+
+\image https://files.seeedstudio.com/wiki/Seeeduino-XIAO/img/Seeeduino-XIAO-pinout.jpg width=66%
```

Comment:
According to the CI this doesn't work with the old stable version of Doxygen. Maybe just use:

```suggestion
![XIAO pinout](https://files.seeedstudio.com/wiki/Seeeduino-XIAO/img/Seeeduino-XIAO-pinout.jpg)
```